### ISSUE TITLE
Static if statements fail to build in specific cases.

### ISSUE THREAD

**Issue by dereli** [rollup/rollup#2134]
Issue description is super simple: static else if statements throws error. check the [REPL](https://rollupjs.org/repl?version=0.58.0&shareable=JTdCJTIybW9kdWxlcyUyMiUzQSU1QiU3QiUyMm5hbWUlMjIlM0ElMjJtYWluLmpzJTIyJTJDJTIyY29kZSUyMiUzQSUyMmlmJTIwKHZhbHVlJTIwJTNEJTNEJTNEJTIwMSklMjAlN0IlNUNuJTVDdGZuMSgpJTNCJTVDbiU3RCUyMGVsc2UlMjBpZiglMjAxMiUyMCElM0QlM0QlMjAxMiklMjAlN0IlNUNuJTVDdGZuMigpJTNCJTVDdCU1Q24lN0QlMjIlN0QlNUQlMkMlMjJvcHRpb25zJTIyJTNBJTdCJTIyZm9ybWF0JTIyJTNBJTIyaWlmZSUyMiUyQyUyMm5hbWUlMjIlM0ElMjJteUJ1bmRsZSUyMiUyQyUyMmdsb2JhbHMlMjIlM0ElN0IlN0QlMkMlMjJhbWQlMjIlM0ElN0IlMjJpZCUyMiUzQSUyMiUyMiU3RCU3RCUyQyUyMmV4YW1wbGUlMjIlM0FudWxsJTdE).

	TypeError: Cannot read property 'start' of null
	    at IfStatement.render (/project/node_modules/rollup/dist/rollup.js:15196:52)
	    at IfStatement.NodeBase.render (/project/node_modules/rollup/dist/rollup.js:13610:23)
	    at IfStatement.render (/project/node_modules/rollup/dist/rollup.js:15191:37)
	    at renderStatementList (/project/node_modules/rollup/dist/rollup.js:13702:25)

**Comment by manucorporat** [comment]
Same here:
```
TypeError: Cannot read property 'start' of null at IfStatement.render
           (/Users/manuelmartinez-almeida/repos/death-rate/node_modules/rollup/dist/rollup.js:15204:52) at
           IfStatement.NodeBase.render
           (/Users/manuelmartinez-almeida/repos/death-rate/node_modules/rollup/dist/rollup.js:13611:23) at
           IfStatement.render
           (/Users/manuelmartinez-almeida/repos/death-rate/node_modules/rollup/dist/rollup.js:15199:37) at
           renderStatementList
           (/Users/manuelmartinez-almeida/repos/death-rate/node_modules/rollup/dist/rollup.js:13709:25) at
           BlockStatement.render
           (/Users/manuelmartinez-almeida/repos/death-rate/node_modules/rollup/dist/rollup.js:14500:13) at
           IfStatement.NodeBase.render
           (/Users/manuelmartinez-almeida/repos/death-rate/node_modules/rollup/dist/rollup.js:13611:23) at
           IfStatement.render
           (/Users/manuelmartinez-almeida/repos/death-rate/node_modules/rollup/dist/rollup.js:15199:37) at
           renderStatementList
           (/Users/manuelmartinez-almeida/repos/death-rate/node_modules/rollup/dist/rollup.js:13709:25) at
           BlockStatement.render
           (/Users/manuelmartinez-almeida/repos/death-rate/node_modules/rollup/dist/rollup.js:14500:13) at
           FunctionExpression.NodeBase.render
           (/Users/manuelmartinez-almeida/repos/death-rate/node_modules/rollup/dist/rollup.js:13611:23)
```

**Comment by lukastaegert** [comment]
Thanks for posting this and sorry for the inconvenience. Fix at #2146

**Comment by dereli** [comment]
@lukastaegert, thanks for quick action and for all the efforts.
